lib/web_ui/lib/src/engine/canvaskit/painting.dart

```diff
@@ -273,29 +193,16 @@ class CkPaint implements ui.Paint {
     if (_imageFilter == value) {
       return;
     }
-    final CkManagedSkImageFilterConvertible? filter;
+
     if (value is ui.ColorFilter) {
-      filter = createCkColorFilter(value as EngineColorFilter);
+      _imageFilter = createCkColorFilter(value as EngineColorFilter);
```

Comment:
Perhaps it would be more logical to do this at the point of converstion, rather than in the setter. This happening in the setter always felt kind of weird to me anyway...

Reply:
Good catch! The actual conversion to `SkImageFilter` happens in `CkManagedSkImageFilterConvertible.imageFilter` used in `toSkPaint()` above. But now that I read the various implementations of `CkManagedSkImageFilterConvertible.imageFilter`, they all use `UniqueRef`, so they are subject to GC. I'm going to have to fix that.

Reply:
Done.